openedx/features/wikimedia_features/meta_translations/models.py

```diff
@@ -16,6 +20,16 @@ class CourseBlock(models.Model):
     block_type = models.CharField(max_length=255)
     course_id = CourseKeyField(max_length=255, db_index=True)
 
+    @classmethod
+    def create_course_block_from_dict(cls, block_data, course_id, create_block_data=True):
```

Comment:
why we have marked this method as `classmethod` ?

Reply:
this method is kind of constructor that's why we declared it on class level so that we can call it as follows:

Reply:
in that case, can we make it static?

Reply:
we need `Class` itself as parameter so that we can call it as follows:
`CourseBlock.create_course_block_from_dict(block_data,course_id)` 

otherwise if we make this static we have to send Class as parameter explicitly as shown below:
`CourseBlock.create_course_block_from_dict(CourseBlock, block_data,course_id)`